cardano-chain-gen/test/Test/Cardano/Db/Mock/Unit.hs

```diff
@@ -819,24 +819,26 @@ rollbackBoundary =
   where
     testLabel = "rollbackBoundary"
 
-singleMirCertMultiOut :: IOManager -> [(Text, Text)] -> Assertion
-singleMirCertMultiOut =
+singleMIRCertMultiOut :: IOManager -> [(Text, Text)] -> Assertion
+singleMIRCertMultiOut =
     withFullConfig "config" testLabel $ \interpreter mockServer dbSync -> do
       startDBSync dbSync
 
       _ <- withAlonzoFindLeaderAndSubmitTx interpreter mockServer $ \_ ->
             Alonzo.mkDCertTx [DCertMir $ MIRCert ReservesMIR (SendToOppositePotMIR (Coin 100000))] (Wdrl mempty)
 
+      a <- fillUntilNextEpoch interpreter mockServer
```

Comment:
For configs that fork directly to Shelley, I'm pretty sure is some underflow on the first epoch. Things are normal after epoch 0. I've opened a ticket for Epoch 0. https://github.com/input-output-hk/cardano-db-sync/issues/1054